src/lib/operators/join_sort_merge.cpp

```diff
@@ -441,11 +441,11 @@ class JoinSortMerge::JoinSortMergeImpl : public AbstractJoinOperatorImpl {
   }
 
   /**
-  * Adds the rows without matches for left outer joins for non-equi operators (<, <=, >, >=).
+  * Adds the rows without matches for right outer joins for non-equi operators (<, <=, >, >=).
   * This method adds those rows from the left table to the output that do not find a join partner.
   * The outer join for the equality operator is handled in _join_runs instead.
   **/
-  void _left_outer_non_equi_join() {
+  void _right_outer_non_equi_join() {
```

Comment:
Srsly? 😂

Reply:
The code read as if it would be doing the exact opposite of what the name implied, so I switched the name and the tests passed. Didn't go deeper, though